assets/src/less/settings.less

```diff
@@ -36,8 +36,8 @@
 
     .dokan-banner {
       border: 4px dashed #d8d8d8;
-      height: 300px;
-      margin: 0 0 30px 35px;
+      min-height: 300px;
```

Comment:
only removing the height property will solve the issue since we are overriding this value from `templates/settings/store-form.php` file, there is no need to add min-height property.
Also setting left+right to auto for margin property will center the image. Since our setting form start from letf side, this will create design inconsistency. I think this should be changed from the theme end.